api/msi.cc

```diff
@@ -39,23 +38,20 @@ void synchronizeFieldData_parasol(apf::FieldDataOf<T>* data, apf::Sharing* shr,
     shr = getSharing(m);
     delete_shr=true;
   }
+  int own_plane = PCU_Comm_Self()%8;
```

Comment:
If the value 8 is truly not a run-time value it should still be either a macro value or a compile-time constant variable.

Reply:
I don't understand what is this doing here and I can't see the whole code.

This was commented out and was an alternative to the current approach as described in the intro to this pull request.

Reply:
@atruszkowska click the 'files changed(4)' tab below and left of the PR title to see the comments inline with the code.

Reply:
@cwsmith thanks, I figured that out but I just wanted to see the whole file quickly and couldn't do it fast.

Reply:
Please remove the two hardcoded values so this can be merged.

Reply:
@cwsmith ok, but if I do so it will create one of those small extra commits that unnecessarily increase the commit history, right? Should I do something about that?

Reply:
@atruszkowska I'm fine with a rebase or an extra commit.

Reply:
@cwsmith I've updated my fork, should I create another pull request? Or is there a way to include the changes within this one? For some reason I thought it will update automatically.

Reply:
@atruszkowska the commits need to be in the `aggy-fusion-clean` branch to automatically update the PR.